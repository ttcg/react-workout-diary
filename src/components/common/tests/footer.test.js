import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../footer';

it('renders without crashing', () => {    

    const component = shallow(<Footer />);

    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});