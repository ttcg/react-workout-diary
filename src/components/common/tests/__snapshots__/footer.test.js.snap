// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<footer
  className="footer"
>
  <Container
    tag="div"
  >
    <span
      className="text-muted float-right"
    >
      v 
      0.10.0
    </span>
    <span
      className="text-muted"
    >
      Â© 
      <t
        ago={false}
        calendar={false}
        decimal={false}
        element={null}
        filter={[Function]}
        format="YYYY"
        fromNow={false}
        interval={60000}
        local={false}
        onChange={[Function]}
        titleFormat=""
        toNow={false}
        unit={null}
        unix={false}
        utc={false}
        withTitle={false}
      />
       TTCG
    </span>
  </Container>
</footer>
`;
